src/shadydom.js

```diff
@@ -82,7 +84,16 @@ if (utils.settings.inUse) {
     // access that requires Shadow DOM behavior to be proxied via `ShadyDOM.wrap`.
     'noPatch': utils.settings.noPatch,
     'nativeMethods': nativeMethods,
-    'nativeTree': nativeTree
+    'nativeTree': nativeTree,
+    'attachDom': (fragment, host, options) => {
```

Comment:
Suggestion: eliminate `attachDom` and instead have attachShadow take another option, such that this pattern triggers the same logic:
```js
this.attachShadow({mode: 'open', shadyUseFragment: fragment});
this.shadowRoot.appendChild(fragment);
```